libraries/TeamSpeak3/TeamSpeak3.php

```diff
@@ -302,6 +303,8 @@ class TeamSpeak3
    *
    * === URI Examples ===
    *   - serverquery://127.0.0.1:10011/
+   *   - serverquery://127.0.0.1:10011/?ssh=1
+   *   - serverquery://127.0.0.1:10011/?ssh=1&server_port=9987
```

Comment:
Needs to specify ssh port instead [[ref](https://github.com/planetteamspeak/ts3phpframework/blob/db3a841662a8cc689992dad28c07525f9dc5a7b6/README.md#ssh-connections-teamspeak-server-only)]

Note difference in port for teaspeak vs teamspeak:
_TeamSpeak_
`$uri = "serverquery://username:password@[fe80::250:56ff:fe16:1447]:10022/?ssh=1";`

_TeaSpeak_
`$uri = "serverquery://username:password@[fe80::250:56ff:fe16:1447]:10011/?tls=1";`